lib/parser/source/tree_rewriter/action.rb

```diff
@@ -46,10 +46,52 @@ def ordered_replacements
         reps
       end
 
+      def nested_actions
+        reps = []
+        reps << [:wrap, @range, @insert_before, @insert_after] unless @insert_before.empty? &&
```

Comment:
nit: could you invert `unless a && b` into `if` please? It's a bit hard to follow

Reply:
Sure